core/src/vmware/vadp_dumper/bareos_vadp_dumper.cc

```diff
@@ -1440,7 +1461,10 @@ static inline bool process_cbt(const char* key, vec allocated, json_t* cbt)
       if (boffset < start_offset + offset_length
           && boffset + blength > start_offset) {
         uint64 offset = std::max(boffset, start_offset);
-        uint64 min_length = std::min(blength, offset_length);
+        uint64 bend = boffset + blength;
+        uint64 oend = boffset + offset_length;
```

Comment:
```suggestion
        uint64 oend = start_offset + offset_length;
```

Reply:
That was probably a copy&paste error from the line above.  Good catch!